### DISCUSSION TITLE
Freeplay song dont open

### DISCUSSION THREAD

**Discussion by 999Karania** [Leather128/LeathersFunkinEngine#215] [Q&A]
I already set the data of the song and put the voices an inst files at the songs folder. Also created the freeplay song, but when I click on it, nothing happends.
Someone pls help me with this

**Comment by demogamer** [comment]
Can you send a screenshot of your error?
And did you put the Data files in the song folder?

**Reply by 999Karania** [reply]
Ok, here it is 
![Captura de Tela (700)](https://user-images.githubusercontent.com/101304227/157651587-cde61972-db85-47c0-bcbb-1b9d36f74041.png)
![Captura de Tela (701)](https://user-images.githubusercontent.com/101304227/157651588-18e7a747-ea72-420e-986d-1ff26659fe33.png)
![Captura de Tela (702)](https://user-images.githubusercontent.com/101304227/157651591-e592480d-8659-4ba2-b5fb-47181fe044c3.png)
The freeplay song just dont work, I spam enter and nothing happends

**Reply by demogamer** [reply]
I had the same problem before. But it was due to the fact that I did not have Data files. Check the fingerprints in the freeplaySonglist file to see if the name matches the Data file.

**Reply by 999Karania** [reply]
IT WORKED, thx bro, the the name of the song on the freeplay song list was ''eStremespeed''. Really thx :)
